README.md

```diff
@@ -11,10 +11,14 @@ This repository contains examples of how to use the [Hume API](https://docs.hume
 | [`typescript-streaming-sandbox`](./typescript-streaming-sandbox/README.md) | `language`, `face`, `burst`, `speech` | Typescript | Next.js   | 
 | [`visualization-example`](./visualization-example/example-notebook.ipynb)  | `face`                                | Python     |           |
 
+
 ## [Empathic Voice Interface (EVI)](https://dev.hume.ai/docs/empathic-voice-interface-evi/overview)
 
-| Name                                                                | Language   | Framework  | 
-| ------------------------------------------------------------------- | ---------- | ---------- |
-| [`evi-typescript-example`](/evi-typescript-example/README.md)       | Typescript | Vanilla JS |
-| [`evi-custom-language-model`](/evi-custom-language-model/README.md) | Python     |            |
-| [`meld`](/meld/README.md)                                           | Typescript | React      |
+| Name | Language | Framework | Description |
+| ------------------------------------------------------------------- | ---------- | ---------- | ----------- |
+| [`evi-typescript-example`](/evi-typescript-example/README.md) | Typescript | Vanilla JS | |
+| [`evi-custom-language-model`](/evi-custom-language-model/README.md) | Python | | |
+| [`meld`](/meld/README.md) | Typescript | React | |
+| [`next-app`](https://github.com/HumeAI/empathic-voice-api-js/tree/main/examples/next-app) | Typescript | Next.js | |
```

Comment:
@jerhadf 
These are shell apps within the empathic-voice-api-js monorepo. (The purpose of these shell apps are to test packages published from the monorepo locally)

There is some work to be done to make these into example apps which serve as external developer resources:
1. Update project names to reflect what they are an example of.
2. Update project READMEs to include: descriptions of what the apps are and what they do, env setup instructions, instructions to build/run the projects, and instructions for usage. 
3. Comments throughout the code which act as documentation for the code, explaining various steps is also really good to have— but can be added later based on developer feedback (like when something is called out as unclear)